antiweb.py

```diff
@@ -19,9 +19,9 @@
 
 
 
-#######
+########
```

Comment:
Because the overlining with six # caused the following sphinx warning:
Warning: Title overline too short.